lib/worker/batcher.ex

```diff
@@ -554,13 +551,22 @@ defmodule BorsNG.Worker.Batcher do
     end
   end
 
-  defp are_reviews_passing(reviews, required) do
-    %{"CHANGES_REQUESTED" => failed, "APPROVED" => passed} = reviews
-
-    case {failed, passed} do
-      {failed, 0} when failed > 0 -> :failed
-      {_, approved} when approved >= required -> :sufficient
-      {0, _} -> :insufficient
+  @spec are_reviews_passing(map, Batcher.BorsToml.t) :: :sufficient | :failed | :insufficient
+  defp are_reviews_passing(reviews, toml) do
```

Comment:
considering renaming to `reviews_status` as the current function name sounds like something that returns a boolean

Reply:
Sure, sounds good to me.